Description: Upstream changes introduced in version 0.2.0-1stormos2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 libbsd (0.2.0-1stormos2) unstable; urgency=low
 .
   * Fix undefined symbol MIN in libbsd.so.
 .
 The person named in the Author field signed this changelog entry.
Author: Andrew Stormont (astormont) <andyjstormont@googlemail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- libbsd-0.2.0.orig/src/strlcat.c
+++ libbsd-0.2.0/src/strlcat.c
@@ -19,6 +19,10 @@
 #include <sys/types.h>
 #include <string.h>
 
+#ifdef __sun
+#include <sys/sysmacros.h>
+#endif
+
 /*
  * Appends src to string dst of size siz (unlike strncat, siz is the
  * full size of dst, not space left).  At most siz-1 characters
--- libbsd-0.2.0.orig/src/nlist.c
+++ libbsd-0.2.0/src/nlist.c
@@ -44,6 +44,10 @@ static char sccsid[] = "@(#)nlist.c	8.1 
 #include <string.h>
 #include <unistd.h>
 
+#ifdef __sun
+#include <sys/sysmacros.h>
+#endif
+
 #if !defined(__NO_A_OUT_SUPPORT)
 #define _NLIST_DO_AOUT
 #endif
--- libbsd-0.2.0.orig/src/strtonum.c
+++ libbsd-0.2.0/src/strtonum.c
@@ -19,6 +19,7 @@
 
 #ifndef __sun
 #include <sys/cdefs.h>
+#include <sys/sysmacros.h>
 #endif
 
 __FBSDID("$FreeBSD$");
--- libbsd-0.2.0.orig/src/hash/helper.c
+++ libbsd-0.2.0/src/hash/helper.c
@@ -22,6 +22,10 @@
 
 #include <hashinc>
 
+#ifdef __sun
+#include <sys/sysmacros.h>
+#endif
+
 /* ARGSUSED */
 char *
 HASHEnd(HASH_CTX *ctx, char *buf)
